src/inmanta/ast/statements/call.py

```diff
@@ -40,10 +43,23 @@ class FunctionCall(ReferenceStatement):
         contributes:
     """
 
-    def __init__(self, name: LocatableString, arguments):
-        ReferenceStatement.__init__(self, arguments)
-        self.name = name
-        self.arguments = arguments
+    def __init__(
+        self,
+        name: LocatableString,
+        arguments: List[ExpressionStatement],
+        kwargs: List[Tuple[LocatableString, ExpressionStatement]],
```

Comment:
wouldn't it be beter to call these namedargs?

Reply:
possibly, I picked kwargs because of the familiarity with python kwargs, but I have no strong preference. If I change these to `named_args`, should I also rename the `wrapped_kwargs`?

Reply:
no, I follow your reasoning